Maestro/subscriptions.json

```diff
@@ -1523,8 +1524,7 @@
         //"https://github.com/dotnet/winforms/blob/release/3.1/**/*",
         "https://github.com/dotnet/winforms/blob/release/5.0/**/*",
         //"https://github.com/dotnet/wpf/blob/release/3.1/**/*",
-        "https://github.com/dotnet/wpf/blob/release/5.0/**/*",
-        "https://github.com/Microsoft/msbuild/blob/release/**/*"
```

Comment:
This section I wasn't sure about, but we don't have anything in the `release/**/*` pattern so it was probably useless before.